jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -54,26 +54,53 @@ define([
     PolymerSupport.init();
 
     // dashboard-actions depends on requirejs text plugin
-    require(['./dashboard-actions'], function(DashboardActions) {
+    require(['./dashboard-actions', './dashboard-metadata'], function(DashboardActions, Metadata) {
```

Comment:
Seems a bit odd to me for this module to use `dashboard-metadata`. I would think that should only be used by `dashboard.js`. Was there a specific reason to bring that up a level?